integration-tests/runtime-tests/test/tests/bondedFinance/txBondedFinanceTests.ts

```diff
@@ -90,7 +96,7 @@ describe('tx.bondedFinance Tests', function() {
    * Runs all tx FAILURE tests for the bondedFinance pallet.
    */
   describe('tx.bondedFinance.offer Failure Tests', function () {
-    if (!testConfiguration.enabledTests.offer_bond__failure.enabled)
+    if (testConfiguration.enabledTests.offer_bond__failure.enabled)
```

Comment:
I think this should be an inverted if. Since in the case `enabled == false` we want ignore this test block.
If we want to disable this, rather change the bool in the test_configuration.json to false.

Reply:
Yes, I missed this. Implemented now.